pc_ble_driver_py/ble_driver.py

```diff
@@ -1466,6 +1466,24 @@ def to_c(self):
             char_md.p_sccd_md = self.sccd_md.to_c()
         return char_md
 
+class BLEGattsValue(object):
+    def __init__(self, value=[0], offset=0):
```

Comment:
Having lists can as the default parameters to functions can often be risky, since any change the list will persist between function calls. It is often safer to do: 

````
f(arg=None):
    if arg == None:
        arg = [0]
````

Reply:
Good catch! That was new to me. Updating with None as default parameter.

Reply:
Oh, about writing tests: I'm not sure how to write a test involving ble_gatts_value_set() as I need a conn_handle and a handle for it?